lib/ssl/doc/src/ssl.xml

```diff
@@ -788,6 +788,15 @@ fun(srp, Username :: binary(), UserState :: term()) ->
       </desc>
     </datatype>
 
+    <datatype>
+      <name name="keep_secrets" since="OTP 23.1"/>
```

Comment:
```since``` tags only work with functions. It would be better to document it in the description: "Since OTP 23.2."